packages/gensx-core/src/checkpoint.ts

```diff
@@ -75,6 +75,9 @@ export class CheckpointManager implements CheckpointWriter {
   private consoleBaseUrl: string;
   private printUrl = false;
 
+  private traceId?: string;
+  private executionRunId?: string;
```

Comment:
Fine for now, but this is more global state that we'll have to rework when we move to durable.

Reply:
(not that we won't have to do that anyways, since we're round tripping the entire JSON on every call from memory right now)